ctags/parsers/abc.c

```diff
@@ -1,124 +0,0 @@
-/*
```

Comment:
I'm not sure this is so much of a joke: yes, it's not recording much, but an [ABC file](http://abcnotation.com/) can apparently contain several tunes, and I assume then that a subsequent title starts the new tune, and this parse then provides a TOC, making it somewhat useful.

Reply:
Oh, that ABC! I thought it was [this ABC](https://en.wikipedia.org/wiki/ABC_(programming_language)) and looking for "T" at the beginning of a line made absolutely no sense. So yeah, it's a pretty esoteric language and the parser doesn't do much but when I know what the language is, it makes sense. I'll remove this commit.

Reply:
Oh, I didn't know that other one :)

Reply:
I didn't know either of them, I just searched for "ABC language" and this one appeared. The parser is now back together with a unit test.